startcommand.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo "BEGIN START COMMAND"
+cd search_gov_crawler
+python benchmark.py -f ./test_domains.json
```

Comment:
test_domains.json doesn't exist.  Seems like you could repoint to the file in `tests/search_gov_spiders/crawl-sites-test.json`

Reply:
You are right, we should have it point to that. I had made test_domains.json as a smaller file that would finish for testing locally